src/WebJobs.Script.Analyzers/DiagnosticDescriptors.cs

```diff
@@ -27,5 +27,11 @@ private static DiagnosticDescriptor Create(string id, string title, string messa
                 messageFormat: "Reuse HttpClient instances to avoid holding more connections than necessary. See helplink for more information.",
                 category: Constants.DiagnosticsCategories.Reliability,
                 severity: DiagnosticSeverity.Warning);
+
+        public static DiagnosticDescriptor IllegalFunctionName { get; }
+            = Create(id: "AZF0003", title: "Illegal Function name",
```

Comment:
Which repo needs to be updated to make the https://docs.microsoft.com/azure/azure-functions/errors-diagnostics/sdk-rules/AZF0003 link working?

Reply:
There's an additional PR in [MicrosoftDocs/azure-docs](https://github.com/MicrosoftDocs/azure-docs) that I'll be linking.